buildUtilities/checkTask.m

```diff
@@ -0,0 +1,7 @@
+function checkTask(~)
+% Identify code issues
+        issues = codeIssues('../toolbox');
```

Comment:
I think you must set here the folder containing the toolbox source code. There might be some issues during merge, since the folder is still called utils in the main branch.